providers/certification-client/tox.ini

```diff
@@ -0,0 +1,68 @@
+[tox]
+envlist = py35,py36,py38,py310
+skip_missing_interpreters = true
+skipsdist=True
+
+[testenv]
+commands =
+    pip -q install -e ../../checkbox-ng
+    - {envbindir}/python3 ../resource/manage.py develop
```

Comment:
2 things:

1. Maybe there should be a comment to explain why we run these ("Required because this provider depends on the resource and the base providers", or something like that). This should be the same for all other providers that depend on other providers.
2. In [Tox doc](https://tox.wiki/en/latest/index.html#system-overview), it says:

> starting a command with a single dash character means ignore exit code

Any particular reason it's used here? (and in the other tox.ini for projects depending on other projects)

Reply:
1. Good idea, I'll add inline comments.

2. I think it's not needed in CI. I had to use this trick locally as the repo provider units were conflicting with previous tox invocations (which left developed providers in `/var/tmp`). Good catch